### PR TITLE
Fix the shownet test on big endian platforms.

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

File: plugins/shownet/ShowNetNodeTest.cpp [hunk 1]
```diff
@@ -36,11 +36,21 @@ namespace ola {
 namespace plugin {
 namespace shownet {
 
+
```
Comment (daveol) [inline plugins/shownet/ShowNetNodeTest.cpp]: trailing enter?

Comment (nomis52) [inline plugins/shownet/ShowNetNodeTest.cpp]: Fixed

**Comment by daveol** [comment]
LGTM, if travis is green

File: plugins/shownet/ShowNetNodeTest.cpp [hunk 2]
```diff
@@ -137,51 +146,54 @@ void ShowNetNodeTest::testHandlePacket() {
...
   // add invalid indexBlocks
-  compressed_dmx->indexBlock[0] = 4;
+  compressed_dmx->indexBlock[0] = ToLittleEndian(4);
   OLA_ASSERT_EQ(false, m_node->HandlePacket(&packet, sizeof(packet)));
   OLA_ASSERT_EQ(0, m_handler_called);
 
   // invalid block length
-  compressed_dmx->indexBlock[0] = ShowNetNode::MAGIC_INDEX_OFFSET;
+  compressed_dmx->indexBlock[0] =
+    ToLittleEndian(ShowNetNode::MAGIC_INDEX_OFFSET);
```
Comment (peternewman) [inline plugins/shownet/ShowNetNodeTest.cpp]: Indent by four from compressed above.

File: plugins/shownet/ShowNetNodeTest.cpp [hunk 3]
```diff
@@ -137,51 +146,54 @@ void ShowNetNodeTest::testHandlePacket() {
...
-  compressed_dmx->netSlot[0] = 1;  // universe 0
+  compressed_dmx->netSlot[0] = ToLittleEndian(1);  // universe 0
   OLA_ASSERT_EQ(false, m_node->HandlePacket(&packet, sizeof(packet)));
   OLA_ASSERT_EQ(0, m_handler_called);
 
   // valid block length, but not enough data
   unsigned int header_size = sizeof(packet) - sizeof(packet.data);
-  compressed_dmx->indexBlock[1] = ShowNetNode::MAGIC_INDEX_OFFSET;
+  compressed_dmx->indexBlock[1] =
+    ToLittleEndian(ShowNetNode::MAGIC_INDEX_OFFSET);
```
Comment (peternewman) [inline plugins/shownet/ShowNetNodeTest.cpp]: Again

File: plugins/shownet/ShowNetNodeTest.cpp [hunk 4]
```diff
@@ -137,51 +146,54 @@ void ShowNetNodeTest::testHandlePacket() {
...
+      100 + ShowNetNode::MAGIC_INDEX_OFFSET);
   OLA_ASSERT_EQ(false,
       m_node->HandlePacket(&packet, header_size + sizeof(ENCODED_DATA)));
   OLA_ASSERT_EQ(0, m_handler_called);
 
   // test invalid slot size
-  compressed_dmx->indexBlock[1] = (ShowNetNode::MAGIC_INDEX_OFFSET +
-      sizeof(ENCODED_DATA));
+  compressed_dmx->indexBlock[1] = ToLittleEndian(
+        ShowNetNode::MAGIC_INDEX_OFFSET + sizeof(ENCODED_DATA));
```
Comment (peternewman) [inline plugins/shownet/ShowNetNodeTest.cpp]: De-dent by 2.

Comment (nomis52) [inline plugins/shownet/ShowNetNodeTest.cpp]: Done

File: plugins/shownet/ShowNetNodeTest.cpp (hunk 2 above)
Comment (nomis52) [inline plugins/shownet/ShowNetNodeTest.cpp]: Done

File: plugins/shownet/ShowNetNodeTest.cpp (hunk 3 above)
Comment (nomis52) [inline plugins/shownet/ShowNetNodeTest.cpp]: Done

**Pull request merged**